src/hooks/useClickOutside.ts

```
@@ -1,17 +1,25 @@
-export const useClickOutside = <T extends HTMLElement>(handler: () => void): RefObject<T | null> => {
+export function useClickOutside<T extends HTMLElement>(
+  handler: () => void,
+  extraRefs?: Array<RefObject<HTMLElement | null>>,
+): RefObject<T | null> {
   const ref = useRef<T | null>(null);
 
   useEffect(() => {
     function handleClick(event: MouseEvent) {
-      if (ref.current && !ref.current.contains(event.target as Node)) {
-        handler();
+      if (ref.current && ref.current.contains(event.target as Node)) return;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)